apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -66,6 +69,14 @@ export function ModuleFunction({
             return result;
         },
     });
+    const executeFunctionError = execute.error
+        ? (execute.error as Error).message
+        : false;
+    const allValues = useWatch({ control });
+    const mutationReset = execute.reset;
```

Comment:
Why?

Reply:
But why not just use `execute.reset` directly, why the intermediate variable?

Reply:
To avoid any changes to the `execute` and not to reset itself (just to be safe)

Reply:
You can just pass `execute.reset` as an effect dependency.

Reply:
lint doesn't like it - I will remove it anyway